srcpkgs/visual-studio-code-bin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'visual-studio-code-bin'
+pkgname=visual-studio-code-bin
+version=1.24.1
+revision=1
+only_for_archs="x86_64"
+wrksrc="${pkgname}-${version}"
+homepage="https://code.visualstudio.com"
+depends="alsa-lib nss libnotify GConf cairo python libXtst fontconfig gtk+ libgcc"
```

Comment:
This can be reduced down to `depends=alsa-lib GConf gtk+ libnotify libXtst nss python`